src/measure_theory/borel_space.lean

```diff
@@ -879,7 +879,25 @@ end complete_linear_order
 
 section conditionally_complete_linear_order
 
-variables [conditionally_complete_linear_order α] [second_countable_topology α] [order_topology α]
+variables [conditionally_complete_linear_order α] [order_topology α]
```

Comment:
with the new proof of measurability for intervals, you just need a linear order here. (just use `(h x).measurable_set` in the proofs below).

Reply:
Indeed, I should have seen that :sweat_smile: